FordLabs/retroquest
# Uniform modal styles

## Overview
In this PR I unified the form styles split out the code from the ModalContentsWrapper so that there were less conditionals and overwritten styles.

Connects N/A

### Demo
<img width="583" alt="Screen Shot 2022-03-30 at 9 32 23 AM" src="https://user-images.githubusercontent.com/17144844/160846671-3895f28f-c518-46a5-bd1c-35d3818bfa01.png">
<img width="748" alt="Screen Shot 2022-03-30 at 9 32 33 AM" src="https://user-images.githubusercontent.com/17144844/160846685-e5f441be-5f4e-480e-98f5-b578b3ffd0bc.png">


### Notes
Things that were done:
- [x] Style the feedback modal to be the same styles as the archive retro modal
- [x] Ensure all modals look good on all screen sizes (especially mobile)
- [x] Split up `ModalContentsWrapper`. Now we have `FormTemplate` that is an actual form and has the cancel/submit buttons in it (and no longer conditional), with the styles that match the archive retro button, and then the styles that match the settings modal got moved into the settings component.

## Testing Instructions
* Check out the feedback modal and ensure it still works as expected, and observe that the styles have changed for it
* Check out the archive modal and ensure it is now the same style as the feedback modal
* Check out the settings modal and ensure it still looks the same
* Ensure all modals are fully responsive and look good on mobile devices (feedback, archive retro, settings, expanded though, and expanded action item modals)
